libvast/src/system/partition.cpp

```diff
@@ -580,6 +587,8 @@ active_partition_actor::behavior_type active_partition(
     },
     [=](const expression& expr,
         partition_client_actor client) -> caf::result<atom::done> {
+      // TODO: Add a `partition_synopsis::lookup()` function that we can use
+      // here if we still have a partition synopsis.
```

Comment:
Please elaborate on this comment. It makes sense now, but might not make too much sense as is for the future self.